tests/beaker_tests/cisco_interface/test_interface_private_vlan.rb

```diff
@@ -51,114 +51,91 @@
   preclean_intf:      true,
   sys_def_switchport: true,
   manifest_props:     {
-    # switchport_mode_private_vlan_host:             no default
-    switchport_mode_private_vlan_host_association:  'default',
-    switchport_mode_private_vlan_trunk_promiscuous: 'default',
-    switchport_mode_private_vlan_trunk_secondary:   'default',
-    # TBD: BROKEN. Should support 'default' but sends literal 'default'
-    # instead of doing default behavior.
-    # switchport_private_vlan_trunk_allowed_vlan:     'default',
-    switchport_private_vlan_association_trunk:      'default',
-    switchport_private_vlan_mapping_trunk:          'default',
-    switchport_private_vlan_trunk_native_vlan:      'default',
+    switchport_pvlan_host:               'default',
+    switchport_pvlan_host_association:   'default',
+    switchport_pvlan_mapping:            'default',
+    switchport_pvlan_mapping_trunk:      'default',
+    switchport_pvlan_promiscuous:        'default',
+    switchport_pvlan_trunk_native_vlan:  'default',
+    switchport_pvlan_trunk_allowed_vlan: 'default',
+    switchport_pvlan_trunk_association:  'default',
+    switchport_pvlan_trunk_promiscuous:  'default',
+    switchport_pvlan_trunk_secondary:    'default',
   },
   resource:           {
-    switchport_mode_private_vlan_host:              'disabled',
-    # switchport_mode_private_vlan_host_association: nil,
-    switchport_mode_private_vlan_trunk_promiscuous: 'false',
-    switchport_mode_private_vlan_trunk_secondary:   'false',
-    # switchport_private_vlan_trunk_allowed_vlan:    nil,
-    # switchport_private_vlan_association_trunk:     nil,
-    # switchport_private_vlan_mapping_trunk:         nil,
-    switchport_private_vlan_trunk_native_vlan:      '1',
+    switchport_pvlan_host:               'false',
+    # switchport_pvlan_host_association:  nil,
```

Comment:
No - these nil properties don't show up with puppet resource. I thought it would be helpful to put them here as comments so that future coders would understand why they're not part of the resource hash.